<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AI_settings</class>
 <widget class="QDialog" name="AI_settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>142</width>
    <height>188</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>101</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="s1_label">
        <property name="text">
         <string>Spieler 1: KI</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ki1_cb">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="s2_label">
        <property name="text">
         <string>Spieler 2: KI</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ki2_cb">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="s3_label">
        <property name="text">
         <string>Spieler 3: KI</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ki3_cb">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="s4_label">
        <property name="text">
         <string>Spieler 4: KI</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ki4_cb">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="ok_button">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>150</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ok_button</sender>
   <signal>clicked()</signal>
   <receiver>AI_settings</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>70</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
